test/unit/setup.js

```diff
@@ -0,0 +1,2 @@
+import 'regenerator-runtime/runtime';
```

Comment:
The latest node version is supported generators and async/await

Reply:
thx. fix